blockstore/splitstore/README.md

```diff
@@ -27,9 +27,34 @@ If you intend to use the discard coldstore, your also need to add the following:
     ColdStoreType = "discard"
```

Comment:
Can we add a big warning saying that the usage of `discard` is discouraged unless you absolutely know what you're doing?

Reply:
well, it's not exactly discouraged, but you should definitely know what you are doing!
I'll add something along this lines.

Reply:
Added a warning.